arch/src/aarch64/mod.rs

```diff
@@ -3,11 +3,15 @@
 
 pub mod layout;
 
+use std::cmp::min;
+
 use memory_model::{GuestAddress, GuestMemory};
 
-/// Stub function that needs to be implemented when aarch64 functionality is added.
+/// Returns a Vec of the valid memory addresses for aarch64
+/// See `layout` module for a drawing of the specific memory model for this platform.
```

Comment:
nit: it would be nice if `layout` could be a [hyperlink to the generated documentation](https://rust-lang.github.io/rfcs/1946-intra-rustdoc-links.html). Maybe
 ``[`layout`](arch.aarch64.layout)`` 
would work, I haven't had the chance to test it.

Reply:
Seems that the winner is `[`layout`](layout)`. I tested it with `cargo doc --target aarch64-unknown-linux-musl --open`.